### PR TITLE
gRPC snippets

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
@hsaliak I still need to add README, but the code is here. Can you take a look?

**Comment by tmatsuo** [comment]
The test is failing probably because of unrelated reasons. The added grpc test is passing.

File: tox.ini [hunk 1]
```diff
@@ -21,6 +25,8 @@ commands =
     pep8: --application-import-names=constants,pubsub_utils
     nosetest: nosetests cmdline-pull
     nosetest: nosetests appengine-push/test_deploy.py
+    grpc: pip install -r requirements.txt --find-links https://gapi-pypi.appspot.com/admin/nurpc-dev
```
Comment (hsaliak) [inline tox.ini]: Please note that this is very temporary, we will be publishing the package to pypi.

Comment (tmatsuo) [inline tox.ini]: Ok, I added README.md that describes that requirement. I also fixed the error in the test. It should pass this time.

**Comment by tmatsuo** [comment]
@hsaliak PTAL

File: tox.ini (hunk 1 above)
Comment (tmatsuo) [inline tox.ini]: Done

Comment (tmatsuo) [inline tox.ini]: Oh wait, I will remove this as well.

**Comment by tmatsuo** [comment]
Ok, ready for another review. PTAL

**Comment by hsaliak** [comment]
Thanks! looks good.

**Pull request merged**